common/src/main/resources/less/style.less

```diff
@@ -350,4 +341,37 @@ body {
 
 .wip:hover .wip-warning {
   visibility: visible;
-}
\ No newline at end of file
+}
+
+// The Target Position layout
+@target-grid-responsive-cutoff: 1350px;
```

Comment:
Where does the cutoff number come from.

SUIR used to have these:

```javascript
Responsive.onlyMobile = { minWidth: 320, maxWidth: 767 }
Responsive.onlyTablet = { minWidth: 768, maxWidth: 991 }
Responsive.onlyComputer = { minWidth: 992 }
Responsive.onlyLargeScreen = { minWidth: 1200, maxWidth: 1919 }
Responsive.onlyWidescreen = { minWidth: 1920 }
```
Bootstrap uses:

```css
// Small devices (landscape phones, 576px and up)
@media (min-width: 576px) { ... }

// Medium devices (tablets, 768px and up)
@media (min-width: 768px) { ... }

// Large devices (desktops, 992px and up)
@media (min-width: 992px) { ... }

// Extra large devices (large desktops, 1200px and up)
@media (min-width: 1200px) { ... }
```

Reply:
Since this was specific to the targets grid, I just determined the width of the viewport when the form fields seemed to be getting too small and chose that number.

Reply:
Today testing on my large screen I noted the grid stacking vertically even for a wide enough screen. Perhaps we could use the 992 cutoff

Reply:
When I switch to 992, the RA/DEC inputs seem a bit small:
![image](https://user-images.githubusercontent.com/6035943/95231069-895e4b80-07b7-11eb-8770-806f984b8baa.png)
But, if I reduce the padding that might be OK. 

Are you running `exploreWDS`, or `targetEditorWDS` when you test? A major shortcoming of CSS responsiveness is that media queries only query the entire viewport size, so that will make a big difference.